src/vrm/utils/disposer.ts

```diff
@@ -1,26 +1,32 @@
+// See: https://threejs.org/docs/#manual/en/introduction/How-to-dispose-of-objects
+
 import * as THREE from 'three';
 
-function disposeMaterial(material: any): void {
+function disposeMaterial(material: THREE.Material): void {
   Object.keys(material).forEach((propertyName) => {
-    if (!!material[propertyName] && typeof material[propertyName].dispose === 'function') {
-      material[propertyName].dispose();
+    const value = (material as any)[propertyName];
+    if (value && typeof value === 'object' && value.isTexture) {
```

Comment:
Can we ommit `typeof value === 'object'` ?
```suggestion
    if (value && value.isTexture) {
```

Reply:
might be? I have no clue that it is omittable

Reply:
I think it's okay.
The line of the newer code is doing just like an optional-chaining. I think.

Reply:
I just had no confidence I can index every truthy.
I know I cannot index `undefined` and `null`

Reply:
yes I think I can everything other than `undefined` or `null` , my bad
I need the optional chaining so bad...... 😩